default_app/index.html

```diff
@@ -114,8 +114,7 @@
 <body>
   <script>
     const electron = require('electron');
-    const remote = electron.remote;
-    const shell = electron.shell;
+    const {remote, shell} = electron;
```

Comment:
It totally could and _should_ be. I had text editor saving issue. Thanks for catching it!